### DISCUSSION TITLE
why not replace rending with bgfx or DiligentEngine  to support all platform?

### DISCUSSION THREAD

**Discussion by raojm** [nem0/LumixEngine#1516] [Ideas]
why not replace rending with bgfx or DiligentEngine  to support all platform?

**Comment by nem0** [comment]
1. it's not fun
2. bgfx does not support modern rendering features (e.g. bindless). Diligent is huge mess. Its DX12 implementation is 10x larger than dx12 in Lumix. Whole Diligent is much larger.  

* there's old Lumix versions using bgfx for rendering

**Comment by wuxianggujun** [comment]
When I searched with AI, I saw that the older branch of this game engine used bgfx, but the newer branch has removed it.
